lib/fortnite-api.js

```diff
@@ -225,7 +225,7 @@ class FortniteAPI {
      * @param {String} [options.lang]
      * @returns {Promise}
      */
-    getBattlepassRewards(season = "current", options = {}) {
+    getBattlepassRewards(options = {}, season = "current") {
```

Comment:
I think this change is out of the scope of this pull request - it is also a breaking change.

Reply:
Okay, I have resend the older version.